### ISSUE TITLE
Requests to v1-production-notifier-configs.s3.amazonaws.com - set remote_config_host to nil

### ISSUE THREAD

**Issue by espen** [errbit/errbit#1479]
After upgrading to Airbrake v5 I am seeing requests to v1-production-notifier-configs.s3.amazonaws.com. This is a default setting in Airbrake to use for remote configuration. See https://github.com/airbrake/airbrake-ruby/pull/611

As there is no need for remote configuration from Airbrake.io this setting should be disabled. I tried setting it to nil and after some basic testing it seems to not do the request.

Should we add `config.remote_config_host = nil` as a default setting in https://github.com/errbit/errbit/blob/master/app/views/apps/_configuration_instructions.html.haml ?

**Comment by stevecrozz** [comment]
That sounds good to me. I haven't used the airbrake v5 gem, but this looks like the right thing to do.

**Comment by espen** [comment]
There is a `__remote_configuration` but it is for private use only. https://github.com/airbrake/airbrake-ruby/blob/96cd0a629310a8411413cb5a89b4f70baa952918/lib/airbrake-ruby/config.rb#L135

**Comment by kyrylo** [comment]
`__remote_configuration` was used in release candidates of airbrake v5 but in the stable version it doesn't exist anymore.

**Comment by javierav** [comment]
Using airbrake-ruby 5.1.1, if you set `nil` to `remote_config_host` you get the following error:

```
undefined method `chomp' for nil:NilClass
**Airbrake: applying remote settings: {}
```

Possible solutions:

1) Implement a config option to disable remote config. Should be implemented by airbrake guys.
2) Implement the remote config endpoint in Errbit and configure the client with `remote_config_host` pointing to our errbit.

**Comment by espen** [comment]
Yeah I already mentioned this to [Airbrake](https://github.com/airbrake/airbrake-ruby/pull/611). They really need to disable this feature, it has [privacy issues](https://github.com/airbrake/airbrake-ruby/issues/626) too.

**Comment by javierav** [comment]
I send a PR with a fix for this issue: airbrake/airbrake-ruby#632

**Comment by javierav** [comment]
Meantime, I have applied this patch to my app:

```
class Airbrake::Config::Processor
  def process_remote_configuration
    return unless @config.remote_config_host

    super
  end
end

Airbrake.configure do |c|
  # ...
  c.remote_config_host = nil
end
```

**Comment by 2called-chaos** [comment]
For airbrake-ruby since v5.2.0 (airbrake is at v11 so I guess you were referring to airbrake-ruby v5) the config option is `c.remote_config = false` ([ref](https://github.com/airbrake/airbrake-ruby/pull/636))

Anyone mad if I add this to the wiki? https://github.com/2called-chaos/errbit/wiki/Draft:-Airbrake-v11-config-example-for-Errbit
